docs/what-is-rocm.rst

```diff
@@ -129,6 +129,6 @@ Runtimes
 .. csv-table::
   :header: "Component", "Description"
 
-  "`AMD Common Language Runtime (CLR) <https://github.com/ROCm/clr>`_", "Contains source code for AMD's common language runtimes: :doc:`HIP <hip:index>` and OpenCL"
+  "`AMD Common Language Runtime (CLR) <https://github.com/ROCm/clr>`_", "Contains source code for AMD's `common language runtimes (CLR) <https://rocm.docs.amd.com/projects/HIP/en/latest/understand/amd_clr.html>`_ HIP and OpenCL"
```

Comment:
```suggestion
  ":doc:`AMD Common Language Runtime (CLR) <hip:understand/amd_clr>`", "Contains source code for AMD's common language runtimes: HIP and OpenCL"
```

We should use Intersphinx links. Also, I think it's better to link to the docs directly in the left column.